### DISCUSSION TITLE
`pip install ".[dev, examples]` in CONTRIBUTING.md

### DISCUSSION THREAD

**Discussion by daniellepintz** [Lightning-AI/pytorch-lightning#11457] [General]
In [`CONTRIBUTING.md`](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md#testing)  we say to install the following:
```
python -m pip install ".[dev, examples]"
python -m pip install pre-commit
```

What do `dev` and `examples` refer to?

**Comment by ananthsub** [comment]
https://github.com/PyTorchLightning/pytorch-lightning/tree/master/requirements

I'm not sure why we have `devel` instead of `dev` though. @Borda do you know?

**Reply by daniellepintz** [reply]
Thanks!

**Reply by Borda** [reply]
no special reason, just sound cooler :D

**Comment by daniellepintz** [comment]
Resolved: https://github.com/PyTorchLightning/pytorch-lightning/blob/5914fb748fb53d826ab337fc2484feab9883a104/setup.py#L39-L49
